tests/common/src/tools/memory_test.cpp

```diff
@@ -143,3 +143,25 @@ TEST(MemoryTest, SharedPtrDerivedClass)
 
     EXPECT_EQ(allocator.FreeSize(), allocator.MaxSize());
 }
+
+TEST(MemoryTest, DeferRelease)
+{
+    int dummy = 0x42;
+
+    testing::MockFunction<void(int*)> deleter;
+    EXPECT_CALL(deleter, Call(&dummy)).Times(1);
+    {
+        auto defer = DeferRelease(&dummy, deleter.AsStdFunction());
+    }
+}
+
+TEST(MemoryTest, DeferReleaseNotCalledForNull)
+{
+    using namespace testing;
```

Comment:
Don't use using for one call only.

Reply:
I used it for two cases, but it doesn't matter much